libnmstate/schemas/operational-state.yaml

```diff
@@ -241,6 +254,23 @@ definitions:
                       type: integer
                     max-age:
                       type: integer
+                vlan-filtering:
+                  type: boolean
+                vlan-default-pvid:
+                  type: integer
+                vlans:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      vlan-range-min:
+                        type: integer
+                      vlan-range-max:
+                        type: integer
+                      pvid:
+                        type: boolean
+                      untagged:
+                        type: boolean
```

Comment:
These are duplicated lines. Please create reference to a new definition.

Reply:
Done.